Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/submissions/SubmissionsPage.tsx

```diff
@@ -23,6 +26,19 @@ const SubmissionsPage = () => {
         [ loadTotalSubmissionsCount, totalSubmissionsCount ],
     );
 
+    useEffect(
+        () => {
+            if (!user.isAdmin) {
```

Comment:
There is already a useEffect which calls loadTotalUnprocessedSubmissionsCount in SubmissionGrid.tsx. This causes the query to execute two times.